pulpcore/pulpcore/app/settings.py

```diff
@@ -161,23 +161,26 @@
 
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.9/howto/static-files/
+# https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
 
 # A set of default settings to use if the configuration file in
 # /etc/pulp/ is missing or if it does not have values for every setting
 _DEFAULT_PULP_SETTINGS = {
-    # https://docs.djangoproject.com/en/1.8/ref/settings/#databases
+    # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
     'databases': {
         'default': {
             'ENGINE': 'django.db.backends.sqlite3',
             'NAME': '/var/lib/pulp/sqlite3.db',
             'USER': '',
             'CONN_MAX_AGE': 0,
+            'OPTIONS': {
+                'timeout': 20,
```

Comment:
any reason you chose 20?

Reply:
no reason other than that it was the value Django used as an example of raising the sqlite timeout here: https://docs.djangoproject.com/en/2.0/ref/databases/#database-is-locked-errors